.github/workflows/release.yml

```diff
@@ -1,6 +1,11 @@
 name: Release
 
 on:
+  workflow_dispatch:
+    inputs:
+      tag:
```

Comment:
Where's this `tag` input used?

Reply:
Good catch, this was copied from AvalancheGo

Reply:
Removed